src/net/chat.rs

```diff
@@ -1127,7 +1134,7 @@ impl ConversationP2P {
         };
 
         debug!(
-            "Handshake from {:?} {} public key {:?} expires at {:?}",
+            "Handshake from {:?} {} public key {:?} services 0x{} expires at {:?}",
```

Comment:
This could be a good structured log:

```
debug!("Handling handshake"; 
             "neighbor" => ?self,
             "authentic_msg" => &authentic_msg,
             "public_key" =>  &to_hex(&hanshake_data...),
             "services" => &to_hex(&handshake_data.services.to_be_bytes()),
             "expires_block_height" => handshake_date.expire_block_height,
             "supports_mempool_query" => Self::supports_mempool_query(&handshake_data.services),
);
```

Using more structured logs in the networking code would help with debugging and tracking what's happening in the network stack (note, I'm also suggesting combining that later "supports_mempool_query" log into this log entry). Combining logs where possible would also go a long way towards making the debug logs more generally useful.

Reply:
Yup, done